test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
+        self.server.manager(MGR_CMD_CREATE,
+                            SCHED,
+                            id="TestCommonSched")
         TestInterfaces.setUp(self)
 
     def test_create_successful_scheduler_object(self):
         """
         Creates a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="testCreateSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
```

Comment:
OK, I'm starting to see the point.  A qmgr syntax error when doing a scheduler operation should be a test failure.  I still worry about calling it a test failure when the exception is raised.  There are valid errors that will raise that exception.  If qmgr isn't there.  If qmgr doesn't have the executable bit set.  Things like that will raise an exception.  Assuming that the exception is because qmgr threw a syntax error isn't 100% correct.  @vinodchitrali are you OK with the test being marked as a failure in real error conditions?  If so, I'm fine with changing this back.